<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main_window</class>
 <widget class="QMainWindow" name="Main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ridual</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>5</number>
    </property>
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item>
     <widget class="QSplitter" name="panes_splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="Pane" name="left_pane" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>200</height>
        </size>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="Pane" name="right_pane" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>200</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="messages_layout">
          <property name="spacing">
           <number>5</number>
          </property>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout"/>
    </item>
    <item>
     <widget class="QWidget" name="current_queue_notice" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Next task will be placed in the queue </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="current_queue_id">
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>623</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_execute"/>
    <addaction name="action_view"/>
    <addaction name="action_edit"/>
   </widget>
   <widget class="QMenu" name="menu_settings">
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menu_recursive_fetch">
     <property name="title">
      <string>Recursive fetch</string>
     </property>
     <addaction name="action_recursive_fetch_auto"/>
     <addaction name="action_recursive_fetch_on"/>
     <addaction name="action_recursive_fetch_off"/>
    </widget>
    <addaction name="action_general_settings"/>
    <addaction name="action_hotkeys"/>
    <addaction name="menu_recursive_fetch"/>
   </widget>
   <widget class="QMenu" name="menu_go">
    <property name="title">
     <string>Go</string>
    </property>
    <addaction name="action_go_parent_directory"/>
    <addaction name="action_go_places"/>
    <addaction name="action_go_root"/>
    <addaction name="separator"/>
    <addaction name="action_refresh"/>
   </widget>
   <widget class="QMenu" name="menu_task">
    <property name="title">
     <string>Task</string>
    </property>
    <addaction name="action_copy"/>
    <addaction name="action_move"/>
    <addaction name="action_link"/>
    <addaction name="action_rename"/>
    <addaction name="action_move_to_trash"/>
    <addaction name="action_move_from_trash"/>
    <addaction name="action_remove"/>
    <addaction name="separator"/>
    <addaction name="action_abort_last_created_task"/>
   </widget>
   <widget class="QMenu" name="menu_queue">
    <property name="title">
     <string>Queue</string>
    </property>
    <addaction name="action_queue_choose"/>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <widget class="QMenu" name="menuClipboard">
    <property name="title">
     <string>Clipboard</string>
    </property>
    <addaction name="action_clipboard_copy"/>
    <addaction name="action_cut"/>
    <addaction name="action_paste"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menuClipboard"/>
   <addaction name="menu_task"/>
   <addaction name="menu_go"/>
   <addaction name="menu_queue"/>
   <addaction name="menu_settings"/>
   <addaction name="menu_help"/>
  </widget>
  <action name="action_hotkeys">
   <property name="text">
    <string>Hotkeys</string>
   </property>
  </action>
  <action name="action_go_parent_directory">
   <property name="text">
    <string>Parent directory</string>
   </property>
   <property name="shortcut">
    <string>Backspace</string>
   </property>
  </action>
  <action name="action_go_places">
   <property name="text">
    <string>Places</string>
   </property>
   <property name="shortcut">
    <string>Shift+F1</string>
   </property>
  </action>
  <action name="action_go_root">
   <property name="text">
    <string>Root filesystem</string>
   </property>
   <property name="shortcut">
    <string>Shift+F2</string>
   </property>
  </action>
  <action name="action_execute">
   <property name="text">
    <string>Execute</string>
   </property>
   <property name="shortcut">
    <string>Alt+X</string>
   </property>
  </action>
  <action name="action_view">
   <property name="text">
    <string>View</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="action_edit">
   <property name="text">
    <string>Edit</string>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
  </action>
  <action name="action_copy">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="action_move">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Move</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="action_link">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Link</string>
   </property>
  </action>
  <action name="action_symbolic_link">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Symbolic link</string>
   </property>
  </action>
  <action name="action_rename">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Rename</string>
   </property>
  </action>
  <action name="action_remove">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
   <property name="shortcut">
    <string>Shift+F8</string>
   </property>
  </action>
  <action name="action_move_to_trash">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Move to trash</string>
   </property>
   <property name="shortcut">
    <string>F8</string>
   </property>
  </action>
  <action name="action_refresh">
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="action_general_settings">
   <property name="text">
    <string>General settings</string>
   </property>
  </action>
  <action name="action_recursive_fetch_auto">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Auto</string>
   </property>
  </action>
  <action name="action_recursive_fetch_on">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>On</string>
   </property>
  </action>
  <action name="action_recursive_fetch_off">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Off</string>
   </property>
  </action>
  <action name="action_queue_choose">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Choose a queue for next task</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="action_clipboard_copy">
   <property name="text">
    <string>Copy files</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_cut">
   <property name="text">
    <string>Cut files</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="action_paste">
   <property name="text">
    <string>Paste files</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="action_abort_last_created_task">
   <property name="text">
    <string>Abort last created task</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="action_move_from_trash">
   <property name="text">
    <string>Move from trash</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Pane</class>
   <extends>QWidget</extends>
   <header>Pane.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
